src/pages/Events/Events.layout.vue

```diff
@@ -0,0 +1,28 @@
+<template>
+  <div class="events__header"></div>
+  <div class="events__container">
+    <div class="events__list">
+      <event-card
+        v-for="event in events"
+        :key="event.id"
+        :title="event.title"
+        :image_url="event.image_url || DIDLogo"
+        :date="new Date(event.date)"
+        :is-announcement="event.type === EventType.PUBLIC"
```

Comment:
Just to confirm, I remember the TYPE changed to be 

`type: 'ATTENDANCE' | 'WORKSHOP'`

Please confirm if your code support these new types

Reply:
@Israel-Laguan Yes, those types are supported.

![image](https://user-images.githubusercontent.com/66505715/176553137-e6e09ec8-fc0f-4e94-ae07-86d55bf6e6ec.png)

I used the name "Public" / "Private" for the variables instead, because they're the more general names for that, so we avoid having to change it always that it changes